tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,7 +16,7 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        Gallery(self.marionette).wait_to_be_displayed()
```

Comment:
This is already the Gallery app, right? Then you can do self.wait_to_be_displayed() , I would think.
But this app needs a manifest_url property still.